doc/source/ray-air/examples/torch_incremental_learning.ipynb

```diff
@@ -291,15 +288,13 @@
     "    mnist_dataset = torchvision.datasets.MNIST(\"./data\", download=True, train=train, transform=transform)\n",
     "    mnist_dataset = ray.data.from_torch(mnist_dataset)\n",
     "    \n",
-    "    def convert_batch_to_pandas(batch):\n",
-    "        images = [np.array(item[0]) for item in batch]\n",
-    "        labels = [item[1] for item in batch]\n",
+    "    def convert_batch_to_numpy(batch):\n",
+    "        images = np.array([np.array(item[0]) for item in batch])\n",
```

Comment:
OOC when do we use `np.array` and when do we use `np.concatenate`/`np.stack`?

Reply:
seems np.array and np.stack are equivalent if we are stacking along axis 0.

np.concatenate is used to concat along an existing axis, and not create a new one.